### PR TITLE
Skrite Time (babby's first PR)

### PR DESCRIPTION
**Pull request by TheEliminator16**
Added adminbus-only monster, hits like a bear and is way tougher

### CODE REVIEW

**Comment by Arthurdentist** [comment]
WHERE ARE THE IMAGES

**Comment by Senaat1** [comment]
It's shit also no changelog

**Comment by ghost** [comment]
It better be tested

**Comment by clusterfack** [comment]
Why not just make it a child of bear

File: code/modules/mob/living/simple_animal/hostile/skrite.dm [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+	. =..()
+	if(!.)
+		return
+
+	switch(stance)
+
+		if(HOSTILE_STANCE_TIRED)
+			stop_automated_movement = 1
+			stance_step++
+			if(stance_step >= 10) //rests for 10 ticks
```
Comment (N3X15) [inline code/modules/mob/living/simple_animal/hostile/skrite.dm]: need to reset stance_step here.

File: code/modules/mob/living/simple_animal/hostile/skrite.dm [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+					src.dir = get_dir(src,target)	//Keep staring at the mob
+
+					if(stance_step in list(1,4,7)) //every 3 ticks
+						var/action = pick( list( "screams at [target]", "sharpens its claws while staring at [target]" ) )
+						if(action)
+							emote(action)
+			if(!found_mob)
+				stance_step--
+
+			if(stance_step <= -20) //If we have not found a mob for 20-ish ticks, revert to idle mode
```
Comment (N3X15) [inline code/modules/mob/living/simple_animal/hostile/skrite.dm]: reset stance_step for each of these conditions.

**Comment by Kurfursten** [comment]
I am with @clusterfack why isn't this a child of bear? No need to paste combat AI etc if it is no different

**Comment by TheEliminator16** [comment]
Changed to be a child of bear, removed separate dm.

**Comment by Zth--** [comment]
Screenshots PLEASE

**Comment by TheEliminator16** [comment]
skrite

**Comment by AutismSpider** [comment]
Can you give it better attack text? or are people fine with the smashes into X+no sound.

When they were bus in the sound they made when they hit stuff was fine+the attack text

**Comment by TheEliminator16** [comment]
The smashes is just for beating up glass. The actual attack text is "uses its blades to stab".
The attack sound is the same as was used in the bus.

**Comment by icantthinkofanameritenow** [comment]
Can't you make it actually function differently from a bear? The space panda fills the niche of a space bear recolor just fine, we don't need another one.

**Comment by TheEliminator16** [comment]
I use bear code because it's generic hostile mob code, which is all that is needed.
The issue isn't having a recolored bear, the issue is having interesting looking monsters in the game for adminbus purposes. It just so happens that the bear code is helpful for this. Yes, functionally, it is a resprited bear, but only for the sake of cutting down on added bloat.

**Comment by icantthinkofanameritenow** [comment]
I get that, but what's the point of having it look neat when that's all it can do? You should at least consider giving it some kind of ability to make it a worthy adminbus spawn, otherwise you'd be better off just using bears like the adminbus usually does.

**Comment by TheEliminator16** [comment]
I'd absolutely consider that if I knew anything about coding. It's an interesting monster to use, and yes, it can be done without any code changes, simply with VV changes in game, but it's tedious, so I figure it can't hurt to add a sprite and a few lines of code to the game.

**Comment by icantthinkofanameritenow** [comment]
Then follow the time-honored /vg/ tradition of shouting at a real coder to do it until they crack. 

Did you make this thing up by yourself? If not, write down as many details as you can about what it could do in its origin media and hope it gives the coderbus an idea.

**Comment by TheEliminator16** [comment]
Made it up myself. Maybe I'll yell at N3X15, that worked the last time.

**Comment by ghost** [comment]
More bus stuff is great, good luck.

**Comment by TheEliminator16** [comment]
Added changelog

**Comment by TheEliminator16** [comment]
By the way, no, Skrites aren't from anything, I just decided to sprite a monster cause I figured we needed it and I ended up liking it.

**Comment by Skullyton** [comment]
Who did the skrite sprite?

**Comment by TheEliminator16** [comment]
Gave it a toxin projectile attack, severely nerfed move speed, nerfed heath and damage.
And I made the sprite.

**Comment by ghost** [comment]
Why the revert?

**Comment by TheEliminator16** [comment]
I fucked up the animal.dmi, it was an outdated version

**Pull request merged**